scene/realistic-lighting-and-shadows/README.metadata.json

```
@@ -1,35 +1,38 @@
 {
-  "category": "Scene",
-  "description": "Show realistic lighting and shadows for the time of day.",
-  "ignore": false,
-  "images": [
-    "RealisticLightingAndShadows.png"
-  ],
-  "keywords": [
-    "3D",
-    "lighting",
-    "realism",
-    "realistic",
-    "rendering",
-    "shadows",
-    "sun",
-    "time",
-    "ArcGISScene",
-    "LightingMode",
-    "SceneView.setSunLighting",
-    "SceneView.setSunTime"
-  ],
-  "redirect_from": "",
-  "relevant_apis": [
-    "ArcGISScene",
-    "LightingMode",
-    "SceneView.setSunLighting",
-    "SceneView.setSunTime"
-  ],
-  "snippets": [
-    "src/main/java/com.esri.samples.realistic_lighting_and_shadows/RealisticLightAndShadowsSample.java",
-    "src/main/java/com.esri.samples.realistic_lighting_and_shadows/RealisticLightAndShadowsController.java",
-    "src/main/resources/realistic_light_and_shadows/main.fxml"
-  ],
-  "title": "Realistic lighting and shadows"
+    "category": "Scene",
+    "description": "Show realistic lighting and shadows for the time of day.",
+    "ignore": false,
+    "images": [
+        "RealisticLightingAndShadows.png"
+    ],
+    "keywords": [
+        "3D",
+        "lighting",
+        "realism",
+        "realistic",
+        "rendering",
+        "shadows",
+        "sun",
+        "time",
+        "ArcGISScene",
+        "LightingMode",
+        "SceneView.setSunLighting",
+        "SceneView.setSunTime"
+    ],
+    "redirect_from": [
+        ""
+    ],
```

it looks like the script has made an array with an empty string here: @JonLavi this was one of the formats that breaks the website. Is there something in the script that needs to be updated to catch this? The correct format to use in the instance of there not being a redirect from is just an empty string:

```suggestion
    "redirect_from": "",
```

---

Yep the `"redirect from": [""]` shouldn't be happening and I am not entirely sure why it did.
Possibly because the previous file had `""` for this value, and the script decided to shove it in an array.
I'll investigate when I get a chance.